test/steps/image_registry.go

```diff
@@ -114,13 +134,13 @@ func getKogitoImageTag(projectLocation, mavenProfiles string) (string, error) {
 		runtimeApplicationImageVersion = "latest"
 	}
 
-	kogitoImageName := getKogitoImageName(projectLocation, mavenProfiles)
+	kogitoImageName := getKogitoImageName(projectLocation, mavenProfiles, options)
 	buildImageTag := fmt.Sprintf("%s/%s/%s:%s", runtimeApplicationImageRegistry, runtimeApplicationImageNamespace, kogitoImageName, runtimeApplicationImageVersion)
 	return buildImageTag, nil
 }
 
-// Returns Kogito image name in the form of "<project location base>(-<maven profile>)*"
-func getKogitoImageName(projectLocation, mavenProfiles string) string {
+// Returns Kogito image name in the form of "<project location base>(-<maven profile>)*(-<option>)*"
+func getKogitoImageName(projectLocation, mavenProfiles string, options []string) string {
```

Comment:
Please call them mavenOptions ?

Reply:
OK, but inside MavenCommand they can be still called Options, right? That would probably make sense. Wdyt?

Reply:
yes, for MavenCommand that can stay as `Options`
for the rest, options is not explicit enough to be Maven related, that was my suggestion :)